src/coreclr/pal/src/map/virtual.cpp

```diff
@@ -509,6 +510,11 @@ static LPVOID VIRTUALReserveMemory(
              flAllocationType |= MEM_RESERVE_EXECUTABLE;
         }
         pRetVal = ReserveVirtualMemory(pthrCurrent, (LPVOID)StartBoundary, MemSize, flAllocationType);
+
+        if (newMemory != NULL)
```

Comment:
Can you please also set the newMemory to false when we are not getting a new memory instead of relying on the caller to have it preset to false?
As a nit, it would also be nice to set the newMemory only when the reservation was successful.

Reply:
Thanks, and done.